### DISCUSSION TITLE
Bootstrapper not installing MSI with Obsolete appearing in install log

### DISCUSSION THREAD

**Discussion by PaulH-HBK** [wixtoolset/issues#8796] [Questions]
We are trying to develop a bootstrapped installer for our application.  We have the MSI which seems to work fine so we have bootstrapped this alongside .NET, ASP .NET and WebView2 installers.  On certain machines we are seeing that the bootstrapped MSI is not being installed which seems to be because it is being flagged as Obsolete in the install log.  Here is the line from the install log that I think is the problem:

Detected package: Advantage.msi, state: Obsolete, cached: No, install registration state: Absent, cache registration state: Absent

I have tried looking in the documentation to see what Obsolete is supposed to mean to try and solve the problem but have had no luck.  I would appreciate any help anyone could give.

**Comment by robmen** [comment]
Which documentation were you looking at?

**Reply by PaulH-HBK** [reply]
Hi Rob and thanks for your response.
I have mainly been looking at https://wixtoolset.org/docs/intro/.  I have also done many a Google search and am struggling to find any information about Wix burn log files containing packages reported as Obsolete.
Does this mean there is an old version caches somewhere?
Does it mean that an old version is still being detected on the target machine?
Does it mean there is a problem with the bootstrapped installer containing the MSI?
All information would be appreciated.
Best regards
Paul

**Comment by robmen** [comment]
`Obsolete` means the package in your bundle has a lower version than the package already installed on the machine.

**Reply by PaulH-HBK** [reply]
Thanks Rob, that makes sense and is sort of what I was expecting.  Firstly is this documented somewhere?  Secondly do you know what it is using to decide that as I am unsure how it can be the case?  My test machine is Windows so I am assuming it is getting some version information from the registry?
